src/components/dictionaryConcepts/components/ConceptDropdown.jsx

```diff
@@ -15,10 +15,10 @@ const ConceptDropdown = props => (
         Add existing concepts
       </button>
       <div className="dropdown-menu" aria-labelledby="dropdownMenuButton">
-        <Link className="dropdown-item" to={`${props.pathName}/import`}>
+        <Link className="dropdown-item" to={`/import${props.pathName}`}>
           Add CIEL concepts
         </Link>
-        <Link to={`${props.pathName}/bulk`} className="dropdown-item">
+        <Link to={`/bulk${props.pathName}`} className="dropdown-item">
```

Comment:
There should be a back slash after bulk?

Reply:
There shouldn't be. The backslash is already there in the pathname.